webrtc.html

```diff
@@ -509,33 +509,42 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
+          </li>
+
+          <li>
+            <p>Append an <code>operation</code> to
+            the <var>operations</var> queue, where
+            an <code>operation</code> represents the asynchronous
+            execution of the called function that will fulfill the
+            promise returned by the function.</p>.
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>If there is exactly one <code>operation</code> in the
+            queue, start the operation.</p>
           </li>
 
           <li>
-            <p>When the asynchronous operation completes (either successfully
-            or with an error), remove the corresponding object from the
-            <code>operations</code> array. After removal, if the array is
-            non-empty, execute the first object queued asynchronously and
-            repeat this step on completion.</p>
+            <p>When any <code>operation</code> completes and fulfills
+            the corresponding promise, remove it from the front of the
+            queue and start the next operation in queue if the queue
+            is non-empty.</p>.
           </li>
```

Comment:
It's not specific enough, so I can't call you on your claims that it works. Show me some code that you claim comes from this, and I can critique it. - The existing old wording hasn't worked in years. It was conceived back when all the methods returned void and basically no-one cared that error handling was totally busted.